app/controllers/application_controller.rb

```diff
@@ -23,7 +23,7 @@ def with_current_request
   end
 
   def current_tenant(current_user)
-    Tenant.find_or_create_by(:external_tenant => current_user.tenant)
+    Tenant.find_or_create_by(:external_tenant => current_user.tenant.to_i)
```

Comment:
@syncrou Do you want to keep this around?

Reply:
@mkanoor my understanding as of Friday was to keep this in here and do a followup PR with the migration change ( and subsequent removal of this. )

Reply:
👍